tasks/setup_environment.yml

```diff
@@ -4,6 +4,10 @@
     project: "{{ group_names[0] }}"
     service_identifier: "{{ group_names[0] }}{{ '' if type in default_node_types else '-' + type }}"
 
+- name: Set architecture variable based on system architecture
+  ansible.builtin.set_fact:
+    system_architecture: "{{ architecture_map[ansible_architecture] | default(ansible_architecture) }}"
```

Comment:
Could you please replace this in other playbooks to like promtail telegraf anda ywhere else if present

Reply:
done